data/pathfinder/paizo/roleplaying_game/core_rulebook/cr_abilities_class.lst

```diff
@@ -83,7 +83,7 @@ Fighter			CATEGORY:Class		DEFINE:Fighter_CFP_Level|0														ABILITY:Intern
 Monk				CATEGORY:Class		DEFINE:Monk_CFP_Level|0	DEFINE:Pool_Monk_Class_Selection|0									ABILITY:Class|AUTOMATIC|Monk ~ Standard Class|!PREABILITY:1,CATEGORY=CLASS,TYPE.AltMonkChoice															BONUS:VAR|Monk_CFP_Level|MonkLVL|TYPE=Base
 Paladin			CATEGORY:Class		DEFINE:Paladin_CFP_Level|0														ABILITY:Internal|AUTOMATIC|Class Skills ~ Paladin|PREVAREQ:Paladin_CF_ClassSkills,0																	BONUS:VAR|Paladin_CFP_Level|PaladinLVL|TYPE=Base
 Ranger			CATEGORY:Class		DEFINE:Ranger_CFP_Level|0														ABILITY:Internal|AUTOMATIC|Class Skills ~ Ranger|PREVAREQ:Ranger_CF_ClassSkills,0																	BONUS:VAR|Ranger_CFP_Level|RangerLVL|TYPE=Base
-Rogue				CATEGORY:Class		DEFINE:Rogue_CFP_Level|0	DEFINE:Pool_Rogue_Class_Selection|0								ABILITY:Internal|AUTOMATIC|Class Skills ~ Rogue|PREVAREQ:Rogue_CF_ClassSkills,0	ABILITY:Class|AUTOMATIC|Rogue ~ Standard Class|!PREABILITY:1,CATEGORY=CLASS,TYPE.AltRogueChoice	BONUS:VAR|Rogue_CFP_Level|RogueLVL|TYPE=Base	BONUS:VAR|RogueSneakAttackLVL|RogueLVL	BONUS:VAR|TrapfindingLVL|RogueLVL	BONUS:VAR|RogueTalentLVL|RogueLVL|PREVARGTEQ:RogueLVL,2	BONUS:VAR|RogueTrapSenseLVL|RogueLVL|PREVARGTEQ:RogueLVL,3	BONUS:VAR|UncannyDodgeFlankingLevel|RogueLVL|TYPE=EachClass.REPLACE|PREVARGTEQ:RogueLVL,4	BONUS:VAR|UncannyDodgeLVL|1|PREVARGTEQ:RogueLVL,4|PREVAREQ:RogueUncannyDodge,0	BONUS:VAR|UncannyDodgeLVL|1|PREVARGTEQ:RogueLVL,8|PREVAREQ:RogueImprovedUncannyDodge,0	BONUS:VAR|MasterStrikeLVL|RogueLVL|PREVARGTEQ:RogueLVL,20
+Rogue				CATEGORY:Class		DEFINE:Rogue_CFP_Level|0	DEFINE:Pool_Rogue_Class_Selection|0								ABILITY:Internal|AUTOMATIC|Class Skills ~ Rogue|PREVAREQ:Rogue_CF_ClassSkills,0	ABILITY:Class|AUTOMATIC|Rogue ~ Standard Class|!PREABILITY:1,CATEGORY=CLASS,TYPE.AltRogueChoice	BONUS:VAR|Rogue_CFP_Level|RogueLVL|TYPE=Base	BONUS:VAR|RogueSneakAttackLVL|RogueLVL	BONUS:VAR|TrapfindingLVL|RogueLVL	BONUS:VAR|RogueTalentLVL|RogueLVL|PREVARGTEQ:RogueLVL,2	BONUS:VAR|RogueTrapSenseLVL|RogueLVL|PREVARGTEQ:RogueLVL,3	BONUS:VAR|MasterStrikeLVL|RogueLVL|PREVARGTEQ:RogueLVL,20
 Sorcerer			CATEGORY:Class		DEFINE:Sorcerer_CFP_Level|0														ABILITY:Internal|AUTOMATIC|Class Skills ~ Sorcerer|PREVAREQ:Sorcerer_CF_ClassSkills,0																BONUS:VAR|Sorcerer_CFP_Level|SorcererLVL|TYPE=Base
```

Comment:
If referencing a class feature like this, the naming standard is "Class ~ Class Feature Name". Each class should have its own class features, and then the shared tracker ability can be granted directly from that ability. Make sense?